Packages.props

```diff
@@ -4,6 +4,9 @@
     <GlobalPackageReference Include="Microsoft.CodeAnalysis.NetAnalyzers" Version="6.0.0" />
     <GlobalPackageReference Include="Microsoft.SourceLink.GitHub" Version="1.1.1" />
     <GlobalPackageReference Include="StyleCop.Analyzers" Version="1.2.0-beta.435" />
+    <PackageReference Update="Azure.Storage.Blobs" Version="12.13.0" />
```

Comment:
I would put them to the `3rd party client dependencies` block.

Reply:
Fixed in [bddb8be](https://github.com/testcontainers/testcontainers-dotnet/pull/555/commits/bddb8be4dc91b51a89a704799bbe006d784e4f8c)